.github/workflows/run-archiver.yml

```diff
@@ -78,20 +75,20 @@ jobs:
           path: ${{ matrix.dataset }}_run_summary.json
 
   archive-run-large:
+    if: ${{ github.event_name == 'schedule' || inputs.large_runner }}
     defaults:
       run:
         shell: bash -l {0}
     strategy:
       matrix:
-        dataset:
-          - epacems
+        # Note that we can't pass global env variables to the matrix, so we manually list the datasets here.
+        dataset: ${{ fromJSON(format('[{0}]', '"epacems"' )) }}
```

Comment:
I suppose we could use the same pattern as with the small runners, and have `inputs.large_runner` actually represent a list of large-runner datasets we would like to run. But since we only have one of those for now I think this is fine!

Reply:
I wanted to prevent us from accidentally kicking off large runners for datasets that don't need them. Another way to do this would've been with an input box constrained against a preset list. But think that'll be more worth revisiting when another dataset gets merged in that calls for a large runner.